### ISSUE TITLE
Log locals() in case of exception

### ISSUE THREAD

**Issue by keyvan-hedayati** [getlogbook/logbook#236]
This is very useful which is provided by Django and Flask when they are in debug mode, after digging though docs I couldn't find same functionality

**Comment by vmalloc** [comment]
This can be a nice feature, but I think it should be explicitly activated if needed (otherwise a lot of use cases would become a lot messier).

Pull requests implementing this are very welcome.

**Comment by k1-hedayati** [comment]
Yes it should be optional, sorry but I can't find time to implement this feature. Please see [boltons](https://github.com/mahmoud/boltons) repo, there is a tbutils module that implements this feature.
